cmd/start/main.go

```diff
@@ -35,13 +33,11 @@ type options struct {
 	version                  string
 	prKey                    string
 	prBase                   string
-	gitRefSpec               string
 	httpProxy                string
 	httpsProxy               string
 	noProxy                  string
 	url                      string
 	gitFullRef               string
-	sslVerify                string
```

Comment:
FYI: When refactoring I noticed that these two params are currently unused in the checkout so I removed them.